SubMan/Modules/Auth/Login/LoginViewModel.swift

```diff
@@ -14,7 +14,7 @@ class LoginViewModel {
     var firebaseHelper: FirebaseHelperProtocol?
     var cryptHelper = CryptHelper()
     var nonce: String?
-    var view: LoginViewController = LoginViewController()
```

Comment:
I have a question, if I don't have the view in the view model, how can I trigger something like an alert when there's an error wrt sign in failure?

Reply:
I mentioned revising your data binding. It will answer this question. The code in Mocking Project is there to show you how (I should keep a count of the number of times we come back to this).